test/arbitrage.test.js

```diff
@@ -338,49 +612,164 @@ test('is_dataframe', t => {
   t.true(act == exp)
 })
 
-test.todo('is_dataframe is false for vector')
+test('is_dataframe is false for vector', t => {
+  var v = [ 1,2,3 ]
+  var act = r.is_dataframe(v)
+  var exp = false
+  t.true(r.all(r.is_equal(act, exp)))
+})
 
 test.todo('is_dataframe is false for JSON list of records')
 
-test.todo('rownames 1')
+test('rownames 1', t => {
+  var df = r.dataframe([1,2], [3,4], {rownames:['a','b']})
+  var act = r.rownames(df)
+  var exp = ['a', 'b']
+  t.true(r.all(r.is_equal(act, exp)))
+})
 
-test.todo('rownames fails for non dataframe')
+test('rownames fails for non dataframe', t => {
```

Comment:
When I see "fails" I think "throws error". This is returning `underfined`, which is different to me.